src/components/Nav/index.js

```diff
@@ -25,6 +29,37 @@ const useStyles = makeStyles(theme => ({
 
 export default function Nav() {
   const classes = useStyles();
+  const [isMenuExpanded, setIsMenuExpanded] = React.useState(false);
```

Comment:
Could we import {useState} at the very top? (e.g. `import React, { useState } from 'react'`)

Reply:
Yes can do! 👍